content/actions/managing-issues-and-pull-requests/commenting-on-an-issue-when-a-label-is-added.md

```diff
@@ -30,38 +30,38 @@ In the tutorial, you will first make a workflow file that uses the [`peter-evans
 1. Copy the following YAML contents into your workflow file.
 
     ```yaml copy
-    {% data reusables.actions.actions-not-certified-by-github-comment %}
-
-    {% data reusables.actions.actions-use-sha-pinning-comment %}
-
```

Comment:
Why were these removed?

Reply:
they add this boilerplate:

```yml
# This workflow uses actions that are not certified by GitHub.
# They are provided by a third-party and are governed by
# separate terms of service, privacy policy, and support
# documentation.

# GitHub recommends pinning actions to a commit SHA.
# To get a newer version, you will need to update the SHA.
# You can also reference a tag or branch, but the action may change without warning.
```

which is no longer needed since
1. we aren't using **any** third party actions anymore so don't need disclaimer
2. there's no actions tot apply the "pin your actions" recommendation to so don't need it

so i figured i should remove 'em lol

Reply:
> we aren't using any third party actions *anymore*

(emphasis mine)

That's a great point! Thanks for making this comment. It helps understanding the reasoning for deleting those. Good choice 👍